### PR TITLE
feat(RuiTextField): add labelPosition prop

### PR DESCRIPTION
**Pull request by cleanerzkp**
also add new style for outside label

Closes #(issue_number)

### CODE REVIEW

File: src/components/forms/text-field/RuiTextField.styles.scss [hunk 1]
```diff
@@ -0,0 +1,367 @@
+@use '@/styles/colors.scss' as c;
```
Comment (kelsos) [inline src/components/forms/text-field/RuiTextField.styles.scss:1]: do you still get errors if you have the style in the SFC?

Comment (cleanerzkp) [reply src/components/forms/text-field/RuiTextField.styles.scss:1]: I did, but now style logic is less than 400 so not anymore

Comment (cleanerzkp) [reply src/components/forms/text-field/RuiTextField.styles.scss:1]: So I guess I can move it back to the main component

**Comment by lukicenturi** [comment]
image

image

image

**Comment by lukicenturi** [comment]
image

**Review by lukicenturi — APPROVED** [review]

**Comment by lukicenturi** [comment]
@cleanerzkp please continue with the `filled` version in new PR

**Pull request merged**